include/errno.h

```diff
@@ -69,5 +69,7 @@
 #define EISCONN         106
 #define ENOTCONN        107
 #define ECONNREFUSED    111
+#define EALREADY        114
```

Comment:
Not doing full review, just a comment: as you're adding new definitions in kernel, please also remove the corresponding ones in `libphoenix` - https://github.com/phoenix-rtos/libphoenix/blob/0fbc038c6699bdc184ced7cdc979b194a2474f06/include/errno.h#L53

Reply:
Thanks for mentioning this. Submitted a PR